test/data/navigatormimetypeswithflash.html

```diff
@@ -1,5 +0,0 @@
-<script>
```

Comment:
This test will fail if you delete this file: https://github.com/brave/brave-core/blob/486993e08a90678b9ace969f33800935fab07f60/chromium_src/third_party/blink/renderer/modules/plugins/navigator_pluginstest.cc